src/fr/composants/Chemin-de-navigation/cas-dutilisation.md

```diff
@@ -17,6 +17,7 @@ tags: ['breadcrumbsFR', 'usage']
 permalink: /fr/composants/chemin-de-navigation/
 date: 'git Last Modified'
 ---
+Le chemin de navigation est un composant obligatoire pour les pages Canada.ca.
```

Comment:
GitHub wouldn't let me select line 13, but my comment is for line 13: shouldn't we update all the "De base" tags for required French components to use the translated version of "Canada.ca required" as well? "De base" means "Basic" which is quite different in meaning.

Reply:
I did raise the exact issue in [in the ticket](https://github.com/cds-snc/design-gc-conception/issues/1000#issuecomment-2400062918) and last week we received the proper tag for french (Requis par Canada.ca) :D I'll need to update this PR!

Reply:
We are going with "Requis sur Canada.ca" according to [this comment](https://github.com/cds-snc/gcds-docs/pull/430#issuecomment-2417512006). I've updated the code to reflect this